### PR TITLE
Add /proc/uptime parser

### PR DESCRIPTION
**Pull request by kfiros**

### CODE REVIEW

File: include/pfs/types.hpp [hunk 1]
```diff
@@ -343,6 +343,12 @@ struct module
     bool operator<(const module& rhs) const { return name < rhs.name; }
 };
 
+struct uptime
+{
+    double running_time;
```
Comment (dtrugman) [inline include/pfs/types.hpp:348]: According to the man page, the first value is: "the uptime of the system (including time spent in suspend)"
I think we should use the value "system_time" here.

File: src/parsers/uptime.cpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+uptime parse_uptime_line(const std::string& line)
+{
+    // Some examples:
+    // clang-format off
+    // 522823.22 2059843.84
+    // clang-format on
+
+    enum token
+    {
+        RUNNING_TIME         = 0,
```
Comment (dtrugman) [inline src/parsers/uptime.cpp:33]: Same "RUNNING" -> "SYSTEM"?

File: sample/format.hpp [hunk 3]
```diff
@@ -579,6 +579,14 @@ inline std::ostream& operator<<(std::ostream& out,
     return out;
 }
 
+inline std::ostream& operator<<(std::ostream& out,
+                                const pfs::uptime& system_uptime)
```
Comment (dtrugman) [inline sample/format.hpp:583]: You can just use "uptime" for the variable since the type requires the "pfs" prefix.

**Review by dtrugman — CHANGES_REQUESTED** [review]
Thanks for contributing!
Looks great man, let's just do that minor change and that's it :)

File: sample/format.hpp (hunk 3 above)
Comment (kfiros) [reply sample/format.hpp:583]: Done.

File: include/pfs/types.hpp (hunk 1 above)
Comment (kfiros) [reply include/pfs/types.hpp:348]: Done.

File: src/parsers/uptime.cpp (hunk 2 above)
Comment (kfiros) [reply src/parsers/uptime.cpp:33]: Done.

**Comment by kfiros** [comment]
Hi, I uploaded fixes for the requested issues.

Thanks,
Kfir

**Review by dtrugman — APPROVED** [review]

**Pull request merged**